kimchi/src/circuits/polynomial.rs

```diff
@@ -10,25 +10,25 @@ use ark_poly::{univariate::DensePolynomial, Evaluations, Radix2EvaluationDomain
 #[derive(Clone)]
 pub struct WitnessEvals<F: FftField> {
```

Comment:
I assumed that the structs in this module were only intended to be used within the crate, are they?